From 1c15b8d896229b11e13134c52579a5c9fac4d5f6 Mon Sep 17 00:00:00 2001
From: Stefan Wysocki <stefan.wysocki@sonymobile.com>
Date: Tue, 18 Sep 2012 14:11:20 +0200
Subject: [PATCH 259/283] Fix for truncated sentence in "Find on Page" field

When typing a text in find on page, then deleting it again the
default text in the field "Find on Page" is truncated by
an invisible view.

This commit fixes the problem by setting the visibility of
mMatches view changes to View.GONE instead of View.INVISIBLE.

Change-Id: I9889f22a254cdabb0cbaba72bff4b14158e5c9ff
---
 core/java/android/webkit/FindActionModeCallback.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/java/android/webkit/FindActionModeCallback.java b/core/java/android/webkit/FindActionModeCallback.java
index 1a4ccfa..6a627e1 100644
--- a/core/java/android/webkit/FindActionModeCallback.java
+++ b/core/java/android/webkit/FindActionModeCallback.java
@@ -152,7 +152,7 @@ class FindActionModeCallback implements ActionMode.Callback, TextWatcher,
             mActiveMatchIndex = matchIndex;
             updateMatchesString();
         } else {
-            mMatches.setVisibility(View.INVISIBLE);
+            mMatches.setVisibility(View.GONE);
             mNumberOfMatches = 0;
         }
     }
-- 
1.8.1.1

